src/index.ts

```diff
@@ -536,8 +538,12 @@ export class Bundler {
 
     for (const source of sources) {
       for (const format of formats) {
+        let title = `Bundle ${source.files.join(', ')} in ${format} format`
+        if (target) {
+          title += ` for target ${target}`
+        }
         tasks.push({
-          title: `Bundle ${source.files.join(', ')} in ${format} format`,
```

Comment:
Oops, I've mixed this here.
This would be more related to when you want multiple build targets though. #219

Reply:
\<question\>Any plans for supporting loading proper `{targets:{}}` hash-map depending on the target?\</question\>